tests/unit/helpers/validate-properties-fail-test.js

```diff
@@ -0,0 +1,61 @@
+import { moduleFor } from 'ember-qunit';
```

Comment:
I would prefer that these test be in the other file. We shouldn't have a file that is specific for failing cases